docs/howto/deployment/asgi/uvicorn.txt

```diff
@@ -0,0 +1,37 @@
+==============================
+How to use Django with Uvicorn
+==============================
+
+.. highlight:: bash
+
+Uvicorn_ is an ASGI server based on ``uvloop`` and ``httptools``,
+with an emphasis on speed.
+
+.. _Uvicorn: https://www.uvicorn.org/
+
+
+Installing Uvicorn
+==================
+
+You can install Uvicorn with Pip::
+
+    pip install uvicorn
+
+
+Running Django in Uvicorn
+=========================
+
+When Uvicorn is installed, a ``uvicorn`` command is available which runs
+ASGI applications. Uvicorn needs to be called with the location of a
+module containing a ASGI application object, followed by what the application
+is called (separated by a colon).
+
+For a typical Django project, invoking Uvicorn would look like::
+
+    daphne myproject.asgi:application
```

Comment:
Seems like this should say `uvicorn ...`?

Reply:
Yup!